### PR TITLE
update raft_test TestLeaderElection2AA

### PR DESCRIPTION
**Pull request by schneiderlin**
change expected state to StateCandidate. since implementation of prevote is not required

### CODE REVIEW

**Comment by schneiderlin** [comment]
https://github.com/pingcap-incubator/tinykv/blob/1df74d600cb0c5001f32d2d21d6f89e9744ee3e0/raft/raft_paper_test.go#L184-L187

also prevote

File: raft/raft_test.go [hunk 1]
```diff
@@ -83,7 +83,7 @@ func TestLeaderElection2AA(t *testing.T) {
 		// are returned instead of the votes being ignored.
 		{newNetworkWithConfig(cfg,
 			nil, entsWithConfig(cfg, 1), entsWithConfig(cfg, 1), entsWithConfig(cfg, 1, 1), nil),
-			StateFollower, 1},
+			StateCandidate, 1},
```
Comment (Connor1996) [inline raft/raft_test.go]: how about just to remove this case

**Comment by Connor1996** [comment]
PTAL @NingLin-P

**Comment by NingLin-P** [comment]
https://github.com/pingcap-incubator/tinykv/blob/1df74d600cb0c5001f32d2d21d6f89e9744ee3e0/raft/raft_test.go#L69-L70

Could you also remove these two useless variables?

**Comment by schneiderlin** [comment]
sure

**Review by NingLin-P — APPROVED** [review]
LGTM, thanks!

**Pull request merged**